_linter/markdown-linter-style.rb

```diff
@@ -0,0 +1,33 @@
+#  Include all rules by default
+all
+
+#  Rule Modifications
+#  --------------------------------
+## MD029 mod - tells linter not to fail if order list items don't start with 1
+rule 'MD029', :style => :ordered
+
+#  Exclude these rules
+#  --------------------------------
+## MD001 - Check that Header levels increment by one level at a time.
```

Comment:
Absolutely, I re-organized these rules according to by design and until content/code is re-worked.  I recommend we keep the "bare url" rule (MD034) as being excluded by default -- I don't really agree with that MD check in general and not sure if we would get it to work nicely with yml frontmatter if we ever wanted a URL in there.

Also, added issues for the rules we are temporarily (hopefully) excluding.